bundles/org.openhab.binding.goecharger/src/main/java/org/openhab/binding/goecharger/internal/handler/GoEChargerHandler.java

```diff
@@ -357,13 +380,10 @@ private String getUrl(String type) {
 
     private void sendData(String key, String value) {
         String urlStr = getUrl(GoEChargerBindingConstants.MQTT_URL).replace("%KEY%", key).replace("%VALUE%", value);
-        logger.debug("POST URL = {}", urlStr);
-
         try {
-            ContentResponse contentResponse = httpClient.newRequest(urlStr).method(HttpMethod.POST)
+            ContentResponse contentResponse = httpClient.newRequest(urlStr).method(HttpMethod.GET)
```

Comment:
Last time i used this i think only post worked, do new firmwares support get now? is this backwards compatible to old hardware?